wwwroot/fileupload.html

```diff
@@ -122,6 +184,139 @@ <h1 class="mt-4">File Upload</h1>
         <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js" crossorigin="anonymous"></script>
         <script src="https://cdn.jsdelivr.net/npm/simple-datatables@7.1.2/dist/umd/simple-datatables.min.js" crossorigin="anonymous"></script>
         <script src="js/site.js"></script>
-        <script>getFiles();</script>
+        <script>
```

Comment:
Would it make more sense to move this to a javascript file and include it instead of putting script functions with colliding names in an html file?